src/plugins/intel_gpu/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1506,6 +1506,9 @@ using activation_random_test_params = std::tuple<data_types,
                                                  activation_additional_params,  // additional_params
                                                  padding>;
 
+template <typename T> double get_tolerance(){return 0;}
+template <> double get_tolerance<float>(){return 1e-5;}
+template <> double get_tolerance<FLOAT16>(){return 1e-4;}
```

Comment:
I think we can just use single tolerance of 1e-4. It is quite small...

Reply:
Done.